lib/SILOptimizer/Mandatory/TFUtilities.h

```diff
@@ -122,6 +122,9 @@ SILLocation getUserSourceLocation(SILInstruction *inst);
 // Other stuff
 //===--------------------------------------------------------------------===//
 
+/// Returns true if `inst` represents a stateful graph op.
```

Comment:
[minor] this func signature seems self-evident enough that i'd slightly prefer to remove the comment, to keep things simple.

if we prefer a comment, i'd rather we describe what "stateful" means (e.g. give some examples). 

also i'd prefer to use the phrase "side effecting" (e.g. a send or enqueue op is side effecting, but not "stateful". a queue op itself is stateful.), but i understand stateful might be an established term in TF.

Reply:
Removed the comment.